src/app/component/RelatedProducts/RelatedProducts.style.scss

```diff
@@ -27,7 +27,7 @@
 
     &-List {
         display: grid;
-        grid-template-columns: 1fr 1fr 1fr 1fr;
+        grid-template-columns: 25% 25% 25% 25%;
```

Comment:
Why you changed to percents here? To shorten you can use `grid-template-columns: repeat(4, 1fr)`

Reply:
1fr should be used with caution. Please check tablet reasolution.

Reply:
It was mentioned in the ticket that fr broke on tablet/mobile. That's why it's now using percentage.